### PR TITLE
AEA-1343 tmp fix for benchmark deployment k8s

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

benchmark cloud run fixes

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: benchmark/run_from_branch.sh [hunk 1]
```diff
@@ -10,14 +10,19 @@ cd $TMP_DIR
 echo "Installing the dependencies."
 pip install pipenv
 # this is to install benchmark dependencies
+pipenv --rm
 pipenv install --dev --skip-lock
 # this is to install the AEA in the Pipenv virtual env
-pipenv run pip install --upgrade aea[all]=="0.10.1"
+pipenv run pip install --upgrade .[all] -y
```
Comment (DavidMinarsch) [inline benchmark/run_from_branch.sh:16]: Installing all the development environment is overkill imo.

File: aea/agent_loop.py [hunk 2]
```diff
@@ -210,8 +204,8 @@ def __init__(
 
     def _setup(self) -> None:
         """Set up agent loop before started."""
-        self._skill2skill_message_queue = asyncio.Queue()
         super()._setup()
```
Comment (DavidMinarsch) [inline aea/agent_loop.py:207]: why the different order?

File: benchmark/run_from_branch.sh [hunk 3]
```diff
@@ -10,14 +10,19 @@ cd $TMP_DIR
...
 # this is to install benchmark dependencies
+pipenv --rm
 pipenv install --dev --skip-lock
 # this is to install the AEA in the Pipenv virtual env
-pipenv run pip install --upgrade aea[all]=="0.11.1"
+pipenv run pip install --upgrade .[all] -y
+pipenv run pip uninstall typing -y
+pipenv run pip install aea-ledger-fetchai -y
+pipenv run pip install aea-ledger-cosmos -y
+pipenv run pip install aea-ledger-ethereum -y
```
Comment (marcofavorito) [inline benchmark/run_from_branch.sh:20]: `--no-deps` missing, o/w it will install `aea` from PyPI

File: benchmark/Dockerfile [hunk 4]
```diff
@@ -5,10 +5,11 @@ RUN apk add --no-cache make git bash wget
...
 RUN apk add --update --no-cache gfortran freetype-dev libpng-dev openblas-dev g++ py3-numpy-dev
-RUN apk add --no-cache go
+RUN apk add --no-cache go rust cargo
 
 RUN pip install --upgrade pip pipenv
 RUN pip install aea[all] --upgrade --force-reinstall
 
 RUN wget https://raw.githubusercontent.com/fetchai/agents-aea/main/Pipfile
 RUN pipenv install -d --deploy --skip-lock --system
+RUN pip install aea-ledger-fetchai
```
Comment (marcofavorito) [inline benchmark/Dockerfile:15]: see below, this might be not the right thing to do

Comment (DavidMinarsch) [reply benchmark/Dockerfile:15]: But the question is should we run the benchmark against pypi version or not. I think we should.

File: benchmark/run_from_branch.sh (hunk 3 above)
Comment (solarw) [reply benchmark/run_from_branch.sh:20]: done

File: benchmark/Dockerfile (hunk 4 above)
Comment (solarw) [reply benchmark/Dockerfile:15]: everythin can be updated inside the container later so not a problem, at least i want to be sure dependencies are installed ok

File: benchmark/run_from_branch.sh (hunk 1 above)
Comment (solarw) [reply benchmark/run_from_branch.sh:16]: it's better to have whole environment

File: benchmark/Dockerfile [hunk 5]
```diff
@@ -5,10 +5,13 @@ RUN apk add --no-cache make git bash wget
...
+RUN apk add --no-cache go rust cargo
 
 RUN pip install --upgrade pip pipenv
 RUN pip install aea[all] --upgrade --force-reinstall
 
 RUN wget https://raw.githubusercontent.com/fetchai/agents-aea/main/Pipfile
 RUN pipenv install -d --deploy --skip-lock --system
+RUN pip install--no-deps  aea-ledger-fetchai
+RUN pip install--no-deps  aea-ledger-cosmos
+RUN pip install--no-deps  aea-ledger-ethereum
```
Comment (DavidMinarsch) [inline benchmark/Dockerfile:17]:
```suggestion
RUN pip install --no-deps  aea-ledger-fetchai
RUN pip install --no-deps  aea-ledger-cosmos
RUN pip install --no-deps  aea-ledger-ethereum
```

**Review by marcofavorito — APPROVED** [review]

**Pull request merged**